<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Romulus</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="lblStatus">
      <property name="text">
       <string>&lt;h2&gt;Status: Idle&lt;/h2&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ROM Search</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>991</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h3&gt;Results:&lt;/h3&gt;</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableSearchResults">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>691</width>
          <height>491</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>90</y>
          <width>20</width>
          <height>491</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>90</y>
          <width>171</width>
          <height>191</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="imgCoverSearch">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-3</y>
           <width>171</width>
           <height>191</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="lblSearchDescriptionSelected">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>400</y>
          <width>261</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDownloadSelected">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>540</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>991</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&lt;h3&gt;Title:&lt;/h3&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editSearchTitle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearch">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
           <property name="shortcut">
            <string>Return</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>290</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;b&gt;Title:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSearchTitleSelected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>320</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;b&gt;Year:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSearchYearSelected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>350</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;b&gt;Platform:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSearchPlatformSelected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Local Games Library</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>291</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h2&gt;In your local collection:&lt;/h2&gt;</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableLocalCollection">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>1001</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>10</y>
          <width>441</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;h3&gt;Filter:&lt;/h3&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboLocalFilter"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Multipi World</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h2&gt;Current Activity&lt;/h2&gt;</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableActivity">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>1001</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h2&gt;Find a player&lt;/h2&gt;</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tablePlayersLookup">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>360</y>
          <width>981</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>981</width>
          <height>34</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Game:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editFindGame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Platform:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboFindPlatform"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnFindPlayers">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Find Now</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>My Friends</string>
       </attribute>
       <widget class="QLabel" name="lblFriendsUsername">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>451</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h2&gt;USERNAME&lt;/h2&gt;</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>20</y>
          <width>171</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="lblPictureUser">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-3</y>
           <width>171</width>
           <height>161</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QTableWidget" name="tableFriendsList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>561</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;h2&gt;Friends list&lt;/h2&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;b&gt;Current points:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblFriendsPoints">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;b&gt;Current level:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblFriendsLevel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuRetropie">
    <property name="title">
     <string>Retropie</string>
    </property>
    <addaction name="actionSync_Library"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_Romulus"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRetropie"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSync_Library">
   <property name="text">
    <string>Pi Control Center</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout_Romulus">
   <property name="text">
    <string>About Romulus</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
